### DISCUSSION TITLE
Adding timestamp links (tags)

### DISCUSSION THREAD

**Discussion by albfan** [asciinema/discussions#117] [Ideas]
Say I'm doing an asciicast about config a program. I want to add a tag at some timestamp to say:

1. screen for config file
2. screen for feature A
3. screen for feature B

So you can go directly to that points of screencast

Maybe this is related with asciinema/asciinema#128 (annotations) Is there any docs about that? How can I help to release this feature?

**Comment by ku1ik** [comment]
I'm not sure if I get that right...

You imagine this as a parameter in the URL so you can share link starting at specific place?
Or you're thinking about having a list of these tags somewhere? If so, then where? In the player UI, or below...?

**Comment by albfan** [comment]
Both, sure. links are always useful. but if I share an asciicast I want people to give a quick look to important parts something like:

tag1
  |
 V
tag 2
  |
 V
tag 3


like an expandable right panel.

**Comment by albfan** [comment]
This example web has a "frame jump list":

https://alt.org/nethack/trd/?file=https://s3.amazonaws.com/altorg/ttyrec/tty/2010-12-28.12:38:24.ttyrec.bz2#

**Comment by ku1ik** [comment]
@albfan thanks for the example. This looks like a useful feature indeed.

**Comment by albfan** [comment]
Let me know if I can help to implement it. As you can see some links has unknow frame but others has name "It is written ..." so I think you need to be able to add info to a frame? and to collect that frame jump list

**Comment by bkil** [comment]
Related or duplicate: https://github.com/asciinema/discussions/issues/99

**Comment by ku1ik** [comment]
asciinema player v3.4.0 brings [markers](https://github.com/asciinema/asciinema-player#markers-1), which act as chapter markers, and are displayed as indicators (with label tooltip on hover) on the progress bar.

asciinema.org now has "Markers" field in recording settings form, where markers can be configured.

There's also related new [pauseOnMarkers](https://github.com/asciinema/asciinema-player#pauseonmarkers) option to automatically pause on each enountered marker. If you're using standalone player on your own site then it can be useful in some scenarios.
